Src/Commands/ImportDataFrameCommand.cs

```diff
@@ -77,7 +89,20 @@ protected override void BeginProcessing()
                     return;
                 }
 
-                throw new NotImplementedException();
+                if (ParameterSetName == PsnParquet)
+                {
+                    using FileStream parquetStream = new FileStream(GetUnresolvedProviderPathFromPSPath(ParquetPath), FileMode.Open, FileAccess.Read, FileShare.None);
+                    var df = parquetStream.ReadParquetAsDataFrameAsync().GetAwaiter().GetResult();
+
+                    WriteObject(df);
+                    return;
+                }
+                else

```

Comment:
Keep just `throw new NotImplementedException();` how it was.
In practice it may happen only during development.

Reply:
I wanted to use PS's Terminating error, but ok, will do

Reply:
I don't know, frankly speaking. Just in case, perhaps. Anyway, removed.

Reply:
should I hit "resolve conversation" button below? I do not send PRs that often, I'm not a developer per se, not sure that's is the right way to use these comments and buttons 😄

Reply:
Yes, yes, please resolve if you think it is resolved. This IMHO helps the reviewer (who may verify and unresolve anyway).